app/views/partials/card/payments/no_balance.scala.html

```diff
@@ -25,7 +25,15 @@
 
 <p role="text">@messages("card.you_have_no_tax_to_pay")</p>
 
-<a id="view-ct-statement" href="@statementLink" data-sso="false"
+<p><a id="view-ct-statement" href="@statementLink" data-sso="false"
   data-journey-click="link - click:CT cards:View your CT statement" target="_blank" rel="external noopener">
 @messages("card.view_your_corporation_tax_statement")
 </a>
+</p>
```

Comment:
Do we need the `p` tag around the link? The previous Front End dev had mentioned/requested that we don't wrap links with the `p` tag. :question: